README.md

```diff
@@ -1,107 +1,127 @@
-🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your portfolio. This is intended to be a guideline, feel free to add your own flare to it.
+[![Netlify Status](https://api.netlify.com/api/v1/badges/cd475e07-33f5-4729-80c6-2e2f9f431b14/deploy-status)](https://app.netlify.com/sites/stampdcerts/deploys)
+![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg) ![React](https://img.shields.io/badge/react-v16.8.6-blue) [![Conflux](https://img.shields.io/badge/react--conflux-0.0.2--beta.6-blue)](https://www.npmjs.com/package/react-conflux) ![apollo-boost](https://img.shields.io/badge/apollo--boost-v0.4.3-blue) ![styled-components](https://img.shields.io/badge/styled--components-v4.3.2-ff69b4) [![grommet](https://img.shields.io/badge/design--system-v2.grommet.io-7D4CDB)](https://v2.grommet.io/) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier) ![eslint](https://img.shields.io/badge/eslint-airbnb-hotpink)
 
-🚫 The numbers 1️⃣ through 5️⃣ next to each item represent the week that part of the docs needs to be comepleted by.  Make sure to delete the numbers by the end of Labs.
+#
 
-🚫 Each student has a required minimum number of meaningful PRs each week per the rubric. Contributing to docs does NOT count as a PR to meet your weekly requirements.
+<h1 align="center"><img src="./assets/readme-header.png" /></h1>
 
-# 1️⃣ Title of project goes here
+# Stampd Client
```

Comment:
What do you think about making the deployed site under this section? I feel like it's too low

Reply:
That's a good idea - Updating now

Reply:
Also, I had @byronholmes2018 put the `client` and `server` URLs in their respective repository headers:
![image](https://user-images.githubusercontent.com/28681364/62826572-c195da00-bb72-11e9-8b5a-58e4a9d4af57.png)

Reply:
Ah I see it. I didn't notice it until now. Still, it could be an option. I don't tend to look at the headers usually in docs.